virttest/libvirt_xml/vm_xml.py

```diff
@@ -1001,6 +1001,22 @@ def get_disk_source(vm_name, option="", virsh_instance=base.virsh):
         disks = vmxml.get_disk_all()
         return list(disks.values())
 
+    @staticmethod
+    def get_disk_source_by_expr(vm_name, option="", virsh_instance=base.virsh, exprs="type==file"):
```

Comment:
change get_disk_source_by_expr to get_disk_source_by_reg ? since it is more like one reg.

Reply:
This 'type==file' looks more like a expression, and the name keeps same with function get_disk_all_by_expr as well.